ajax/libs/d3-annotation/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "d3-annotation",
+  "filename": "d3-annotation.min.js",
+  "description": "Use d3-annotation with built-in annotation types, or extend it to make custom annotations. It is made for d3-v4 in SVG.",
+  "keywords": [
+    "d3",
+    "d3-module",
+    "annotation"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/susielu/d3-annotation.git"
+  },
+  "author": {
+    "name": "Susie Lu",
+    "url": "http://www.susielu.com"
+  },
+  "license": "Apache-2.0",
+  "homepage": "http://d3-annotation.susielu.com",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/susielu/d3-annotation.git",
+    "fileMap": [
+      {
+        "basePath": "",
+        "files": [
+          "d3-annotation.js",
+          "d3-annotation.min.js"
```

Comment:
@susielu maybe we can just use something like `d3-annotation*` or `d3-annotation*.+(css|js|map)`?